### DISCUSSION TITLE
I need some card switchers/buttons like...

### DISCUSSION THREAD

**Discussion by sergeynilov** [filamentphp/filament#13346] [Help]
### Package

Form builder

### Package Version

^3.2-stable

### How can we help you?

In laravel 10 / filamentphp 3 app in resource form I need to add several card switchers/buttons,
like :

> a) changing to other status  depending on current status - some buttons can be accessible/not accessible at some time, 
> b) disable some buttons
> c) custom colos for some buttons 
> d) custom icons for some buttons 

Are there some scripts/pugins for this ?

**Comment by leandrocfe** [comment]
https://filamentphp.com/docs/3.x/forms/fields/toggle-buttons
https://filamentphp.com/docs/3.x/forms/actions#overview

**Reply by sergeynilov** [reply]
I added ToggleButtons component with code:

```
ToggleButtons::make('status_ToggleButtons')
    ->label('Toggle status')
    ->options([
    'draft' => 'Draft',
    'scheduled' => 'Scheduled',
    'published' => 'Published'
]),
```

But no controls on the page :

    https://img001.prntscr.com/file/img001/ea_5VcDMS-SLHDoA1wkD8Q.png

Which options are missing ?

**Reply by micraux** [reply]
Does your column name is `status_ToggleButtons`?

**Reply by sergeynilov** [reply]
No in db field is 'status', but if I wrote ` ToggleButtons::make('status') ` - the same empty space

**Reply by micraux** [reply]
Yes, whatever you call, the component shows all the stuff.

I tried your **exact** code above (copy/paste) and it works as expected.

So might be an error elsewhere in your code...
